packages/myst-cli/src/project/toToc.ts

```diff
@@ -2,7 +2,7 @@ import fs from 'node:fs';
 import yaml from 'js-yaml';
 import { join, relative } from 'node:path';
 import { removeExtension } from '../utils/removeExtension.js';
-import type { JupyterBookChapter, TOC } from '../utils/toc.js';
+import type { JupyterBookChapter, TOC } from 'myst-jb-toc';
```

Comment:
Thoughts on naming the package `myst-toc` or something? That might give us room to evolve this function (better support jb-article, and switch between these toc pieces pretty easily in the future).

Reply:
My gut feeling is that we're moving towards not having a ToC as a separate part of MyST in the future development direction. If that's the case, then I think this ToC package will/should always be concerned with reading JB ToCs if needs be. Though, as JB v2 develops, we'd eventually probably move this package out into JB's repo.

Reply:
Ok, fair enough! Maybe we should call it `jupyterbook-toc`, as it is less to do with myst in that scenario.